pkg/apis/pipeline/v1alpha1/pipelinerun_types.go

```diff
@@ -136,6 +136,17 @@ type PipelineRunTaskRunStatus struct {
 	// Status is the TaskRunStatus for the corresponding TaskRun
 	// +optional
 	Status *TaskRunStatus `json:"status,omitempty"`
+	// ConditionChecks maps a conditionCheckName to the Status for the corresponding ConditionCheck
```

Comment:
`conditionCheckName` doesn't seem to be a variable that exists anywhere - maybe more future proof to say something like "maps the name of a condition check"?

Reply:
done!